beacon-chain/p2p/peers/status.go

```diff
@@ -369,6 +369,32 @@ func (p *Status) Connected() []peer.ID {
 	return peers
 }
 
+// Inbound returns the current batch of inbound peers that are connected.
+func (p *Status) Inbound() []peer.ID {
+	p.store.RLock()
```

Comment:
Would it be better to make the critical section shorter? 
```go
p.store.RLock()
peers := p.store.Peers() 
p.store.RUnlock()
for pid, peerData := range peers { }
```

Reply:
The locking follows the general pattern for all the peer handler methods in the package. Given that this is a read only lock, it shouldn't be too bad having it locked this way, if you feel strongly I can change it to the above pattern.

Reply:
No preference from my end. It was just a thought. Sounds good to me then